app/src/main/java/ch/epfl/sdp/peakar/utils/UIUtils.java

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.sdp.peakar.utils;
+
+public class UIUtils {
```

Comment:
Even if the name is quite self-explanatory, should we add a brief description of this class?